lib/includes/ngtcp2/ngtcp2.h

```diff
@@ -4091,7 +4091,8 @@ NGTCP2_EXTERN ngtcp2_ssize ngtcp2_conn_write_stream(
  * function).  Just keep calling `ngtcp2_conn_writev_stream`,
  * `ngtcp2_conn_write_pkt`, or `ngtcp2_conn_writev_datagram` until it
  * returns a positive number (which indicates a complete packet is
- * ready).
+ * ready).  If there is no stream data to include, call
+ * `ngtcp2_conn_write_pkt` to stop coalescing and write a packet.
```

Comment:
I think it would be nice to first recommend ngtcp2_conn_writev_stream with  stream_id = -1 and no stream data, as you described in the above hunk because it is more natural to use same function in a loop.

Reply:
Thank you; updated along these lines.